pallets/fragments/src/lib.rs

```diff
@@ -100,6 +105,15 @@ pub mod pallet {
 	// Dispatchable functions must be annotated with a weight and must return a DispatchResult.
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
+		/// Create a Fragment using an existing Proto-Fragment (only the owner of the Proto-Fragment can call this function and create a Fragment)
+		///
+		/// # Arguements
+		/// * `origin` - The origin of the extrisnic/dispatchable function.
+		/// * `proto_hash` - The hash of the existing Proto-Fragment
+		/// * `metadata` - The metadata (name, external url etc.) of the Fragement that is going to be created
+		/// * `unique` - Whether the Fragment that is being created is unique
+		/// * `mutable` - ¿
```

Comment:
if the item data can be edited (will be more clear when we add the mint/buy/upload bit)